src/System.Management.Automation/engine/CoreAdapter.cs

```diff
@@ -2664,6 +2664,33 @@ internal EventCacheEntry(EventInfo[] events)
             {
                 this.events = events;
             }
+
+            private string _eventDefinition;
+            internal string EventDefinition
+            {
+                get
+                {
+                    if (_eventDefinition == null)
+                    {
+                        var sb = new StringBuilder();
+                        var invokeMethod = new MethodInformation(
+                            events[0].EventHandlerType.GetMethod("Invoke"), parametersToIgnore: 0).method;
+                        sb.AppendFormat(
+                            "event {0} {1}",
+                            ToStringCodeMethods.Type(events[0].EventHandlerType),
+                            events[0].Name);
+
+                        sb.Append("(");
+                        sb.AppendJoin(", ", invokeMethod.GetParameters().Select(p => $"{p.ParameterType.Name} {p.Name}"));
```

Comment:
We think we should explicitly use InvariantCulture for formatting in the two places. It is our common pattern in Engine.

Reply:
`AppendJoin()` doesn't appear to have an overload that takes a format provider, but will do for AppendFormat().

Reply:
I think @iSazonov is referring to the string interpolation used in the `Select` delegate.

Reply:
I meant both - AppendFormat and the string interpolation.